tests/Feature/ValidateCSVTest.php

```diff
@@ -215,6 +215,27 @@ public function test_throws_on_invalid_line(Closure $data): void
         ValidateCSV::dispatch($import);
     }
 
+    public function test_does_not_throw_on_generic_uuid(): void
+    {
+        // Ensure correct columns
+        $line = 'Q4115189$ffa51229-4877-3135-a2e2-a22fe9b7039d' // Emulate non v4 UUID
```

Comment:
yay, a real statement guid from the wild outside world... \o/

Reply:
hehe, also the other ones are super real, they belong to Jane Goodall('s wikidata page)